ADXL362.py

```diff
@@ -156,9 +167,9 @@ def check_all_regs(self):
         return values[2:]
 
     def twos_comp(self,val, bits):
-       ''' Returns two's complement of value given a number of bits
-       '''
-        if( (val&(1<<(bits-1))) != 0 ):
-             val = val - (1<<bits)
-        return val
+       """ Returns two's complement of value given a number of bits
+       """
+       if val&(1<<(bits-1)) != 0:
+           val = val - (1<<bits)
+       return val
```

Comment:
Cool. I'll submit new PRs, since I just bundled a few of them together in the same branch.